### PR TITLE
Package Enter Data Helper

### LINKED ISSUE #26: package enter-data-helper
Package the electron app and provide download for easier use.

### PR DESCRIPTION
**Pull request by benj-zen**
Resolves #26.

Adds the capability of packaging the enter-data-helper app. First release for Linux and MacOS is already up https://github.com/cryptoeng/pqdb/releases.

### CODE REVIEW

File: tools/enter-data-helper/README.md [hunk 1]
```diff
@@ -1,18 +1,52 @@
...
+./enter-data-helper-linux.AppImage /path/to/pqdb
+```
+
+### MacOS
+
+Extract the archive with
+```
+tar -xf enter-data-helper-mac.tar.gz
+```
+
```
Comment (taudor) [inline tools/enter-data-helper/README.md:25]: In order to avoid Apple's Gatekeeper, run

```
xattr -dr com.apple.quarantine "enter-data-helper.app"
```

**Comment by taudor** [comment]
I'm getting an error trying to start the app on a Mac. This error didn't appear yesterday with the other package. I'll try to figure it out.

```
dyld: Library not loaded: @rpath/Electron Framework.framework/Electron Framework
  Referenced from: /Users/tudor/Downloads/./enter-data-helper.app/Contents/MacOS/enter-data-helper
  Reason: no suitable image found.  Did find:
	/Users/tudor/Downloads/enter-data-helper.app/Contents/MacOS/../Frameworks/Electron Framework.framework/Electron Framework: file too short
	/Users/tudor/Downloads/enter-data-helper.app/Contents/MacOS/../Frameworks/Electron Framework.framework/Electron Framework: stat() failed with errno=1
[1]    94301 abort      ./enter-data-helper.app/Contents/MacOS/enter-data-helper
```

**Comment by benj-zen** [comment]
Seems like https://github.com/electron-userland/electron-builder/issues/4135 is the issue. I will adjust the script to manually pack the release.

**Comment by taudor** [comment]
Thanks, works now.

**Pull request merged**